img/gif.cpp

```diff
@@ -0,0 +1,104 @@
+// SPDX-FileCopyrightText: 2023 Robin Lindén <dev@robinlinden.eu>
+//
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include "img/gif.h"
+
+#include <array>
+#include <bit>
```

Comment:
Nothing used from `<array>` and `<bit>`, right?

Reply:
Good catch! They're from an earlier implementation when I'd misread the spec. I'd used std::byteswap to flip width and height from LSB to MSB.